include/enums/cpp_tensor_type.h

```diff
@@ -52,32 +52,32 @@ enum class TensorType{
 //! Helper method to convert between CTensorType and TensorType
 inline TensorType convert_tensor_type(CTensorType type) {
     TensorType t;
-    switch(type) {
-        case CTensorType::c_dbl :
+    switch (type) {
```

Comment:
We should probably add the ``invalid`` type to these switch cases too.  Also, maybe the ``0`` value for TensorType should be ``invalid`` to match the naming convention used in other enums?

Reply:
I'll change it to invalid when I consolidate the enumerations during the error handling work. Cases added to the switch statements